src/config/index.js

```diff
@@ -0,0 +1,5 @@
+const CONFIG = {
+  SHOW_STUDY_LIST: process.env.SHOW_STUDY_LIST || true,
```

Comment:
Won't this always be true?

`undefined || true` --> true
`false || true` --> true

Reply:
You are right, this doesn't work with booleans. I will fix using the runtime approach